City_CRUD_API/schemas.py

```diff
@@ -0,0 +1,38 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
```

Comment:
Consider making `additional_info` optional in the `CityBase` class if it's not always required. This would align with the optional nature of this field in the `CityUpdate` class.